js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,16 @@
+function abbreviateString(str) {
+  if (typeof str !== 'string') {
+    throw new Error('Input must be a string !');
+  }
+
+  const letterSet = str.split(' ');
+  /**
+   * Assumptions:
+   *    Input string must have more than one letter
+   *    Last letter cannot be special character or number
+   *    First character of first letter will be capital in input string
```

Comment:
Why you are making this assumption that the first letter of the string would be capital?
It's not mentioned in the README file